util/validators.js

```diff
@@ -577,6 +596,132 @@ module.exports.validateRegisterAlumniInput = (
   };
 };
 
+module.exports.validateEditAlumniProfileInput = (
+  firstName,
+  lastName,
+  oldEmail,
+  newEmail,
+  undergrad,
+  grad,
+  employer,
+  position,
+  location,
+  linkedin
+) => {
+  const errors = {};
+
+  const nameValidator = /^[a-zA-Z ',.-]{3,20}$/;
+  const emailValidator = /^([0-9a-zA-Z]([-.\w]*[0-9a-zA-Z])*@([0-9a-zA-Z][-\w]*[0-9a-zA-Z]\.)+[a-zA-Z]{2,12})$/;
+  const yearValidator = /^\d{4}$/;
+
+  if (firstName.trim() === "") {
+    errors.firstName = "First name is required.";
+  } else {
+    if (!firstName.match(nameValidator)) {
+      errors.firstName =
+        "First Name must be at least 3 characters, max 20. No special characters or numbers.";
+    }
+  }
+
+  if (lastName.trim() === "") {
```

Comment:
if you could make these changes for all of these, just much cleaner / best practice for js code.
```suggestion
  if (!lastName.trim()) {
```